templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -594,3 +594,23 @@ raise UnexpectedWebhookEventType(webhook_name, event_type)
 
 `webhook_name` is the name of the integration that raises the exception.
 `event_type` is the name of the unexpected webhook event.
+
+
+### Working with a locally hosted third-party service
+
+Some third-party services like Jira and Bitbucket Server can be hosted locally
+and it's possible to allow these services to be able to directly post to your
+Zulip development enviroment. But first you'll need to make sure that your Zulip
+development environment (notably the virtual machine) is visible to the locally
+hosted service. Here's how you can do that:
+
+1. Open zulip/Vagrantfile. Replace the line `host_ip_addr = "127.0.0.1"` with
+`host_ip_addr = "0.0.0.0"`. This means that the VM is now listening on all IPv4
+addresses available on your local machine.
+
+2. Run `export EXTERNAL_HOST="<IP_OF_HOST>:9991"` where <IP_OF_HOST> is your
+host machine's IP address. This can be found by running `ifconfig` outside of
+the Zulip development environment, on your local PC, you should see
+something like `inet 192.168.1.7` for example in one of the lines of output.
+
+3. Pass `--interfaces=''` to `./tools/run-dev.py`.
```

Comment:
Is it worth adding a paragraph after this that briefly mentions using the external IP and port forwarding for externally hosted services?

Reply:
Good question, I'm not sure... My main concern here is giving too light of an answer and confusing people who don't already know this.

For the time being, I'm thinking that it's not really needed, and that we could always add it later if and as required.

@davidtwco Do you want to make the final call on if we should add such a paragraph?

Reply:
If you feel that it isn't needed then we could probably omit it.